framework/drawer.h

```diff
@@ -0,0 +1,207 @@
+/* Copyright (c) 2023, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <array>
+#include <vector>
+#include <string>
+#include <imgui.h>
+#include <stdint.h>
+
+namespace vkb
+{
+class Drawer;
+using HPPDrawer = Drawer;
```

Comment:
I'd prefer to not have that `HPPDrawer` anymore. So would be great, if you could change to `Drawer`, wherever it's used.

Reply:
Done.